docs/plugins.md

```diff
@@ -8,7 +8,13 @@ redirect_from:
  - /docs/usage/modules/
 ---
 
-Out of the box Babel doesn't do anything. You need to use plugins to transform your code.
+Babel is a compiler. At a high level, it has 3 stages that it runs code in: parsing code, transforming code, and code generation (like many other compliers).
```

Comment:
typo: compliers. Personally I don't think the repetition of *code* is necessary (e.g. could be "parsing, transforming, and generation"), but I'm not making an issue of it if you prefer it that way :smiley: